### ISSUE TITLE
Duplicate key warning message does not include the key value for context [rt.cpan.org #73121]

### ISSUE THREAD

**Issue by ingydotnet** [ingydotnet/yaml-pm#119]
https://rt.cpan.org/Ticket/Display.html?id=73121

```
I've found that duplicate keys can be hard to isolate because the
warning message from YAML::Loader does not include the name of the
key. I have attached a trivial patch which adds the capability.
```

**Comment by perlpunk** [comment]
The patch looks good to me. I'll create a PR

**Comment by perlpunk** [comment]
Merged #169

**Comment by perlpunk** [comment]
released 1.20_002
